crates/aws/src/constants.rs

```diff
@@ -99,8 +105,8 @@ pub const S3_OPTS: &[&str] = &[
     AWS_SECRET_ACCESS_KEY,
     AWS_SESSION_TOKEN,
     AWS_S3_LOCKING_PROVIDER,
-    AWS_S3_ASSUME_ROLE_ARN,
-    AWS_S3_ROLE_SESSION_NAME,
```

Comment:
I mentioned to @hntd187 in Slack that I would like to avoid removing these deprecation warnings until we have a few more releases which is print deprecation warnings for people